xml/System.Windows.Threading/Dispatcher.xml

```diff
@@ -1660,6 +1688,12 @@
   
  ]]></format>
         </remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="callback" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException">
+          <paramref name="timeout" /> is a negative number other than -1 milliseconds, which represents an infinite wait.</exception>
```

Comment:
This method also needs the verbiage about negative timeouts throwing in cross-thread calls but not in same-thread calls.